module/Activity/src/Activity/Form/ActivityFieldFieldset.php

```diff
@@ -5,18 +5,20 @@
 use Activity\Model\ActivityField;
 use Zend\Form\Fieldset;
 use Zend\Stdlib\Hydrator\ClassMethods as ClassMethodsHydrator;
+use Doctrine\Common\Persistence\ObjectManager;
+use DoctrineModule\Stdlib\Hydrator\DoctrineObject as DoctrineHydrator;
 use Zend\Validator\NotEmpty;
 use Zend\InputFilter\InputFilterProviderInterface;
 
 class ActivityFieldFieldset extends Fieldset implements InputFilterProviderInterface
 {
-    public function __construct() {
+    public function __construct(ObjectManager $objectManager) {
 
         parent::__construct('activityfield');
 
-        $this->setHydrator(new ClassMethodsHydrator(false))
-             ->setObject(new ActivityField());
-
+        $this->setHydrator(new DoctrineHydrator($objectManager));
```

Comment:
This is not the way to inject a Hydrator. See the `Education` module for a proper example on this.